apps/portal/src/components/debug/onboarding-data-display.tsx

```
@@ -0,0 +1,299 @@
+import { useOnboarding } from "@/hooks/use-onboarding";
+import { clearOnboardingData } from "@/lib/storage/onboarding-storage";
+import { clearAllVendorData, getAllVendorData } from "@/lib/storage/vendor-storage";
+
+interface OnboardingDataDisplayProps {
+  userId: string;
+}
+
+export function OnboardingDataDisplay({ userId }: OnboardingDataDisplayProps) {
+  const { progress, onboardingData, currentStep, isLoading, error, resetOnboarding } = useOnboarding(userId);
+
+  const [vendorData, setVendorData] = useState<any[]>([]);
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)